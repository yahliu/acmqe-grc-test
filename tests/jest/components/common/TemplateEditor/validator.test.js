/*******************************************************************************
 * Licensed Materials - Property of IBM
 * (c) Copyright IBM Corporation 2018, 2019. All Rights Reserved.
 *
 * Note to U.S. Government Users Restricted Rights:
 * Use, duplication or disclosure restricted by GSA ADP Schedule
 * Contract with IBM Corp.
 *******************************************************************************/
/* Copyright (c) 2020 Red Hat, Inc. */
'use strict'
import { validateYAML } from '../../../../../src-web/components/common/TemplateEditor/utils/validate-yaml'

//parsed, exceptions, locale
describe('truncate middle', () => {
  it('should truncate string as expected', () => {
    expect(validateYAML('', [], '')).toMatchSnapshot()
  })
})
