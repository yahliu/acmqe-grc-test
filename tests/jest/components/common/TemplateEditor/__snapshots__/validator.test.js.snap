// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`truncate middle should truncate string as expected 1`] = `undefined`;
